tests/pytests/functional/states/test_ssh_auth.py

```diff
@@ -0,0 +1,62 @@
+import logging
+import pathlib
+
+import pytest
+
+import salt.states.ssh_auth as ssh_auth_state
+import salt.utils.files
+
+log = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.requires_network,
+    pytest.mark.skip_if_binaries_missing("svn"),
```

Comment:
```suggestion
```
Do we really need the subversion binary to run all of these tests?
That binary is never used in the tests.

Reply:
Nice catch, residual from copy/pasting.